python/nammu/controller/EditSettingsController.py

```diff
@@ -0,0 +1,58 @@
+'''
+Copyright 2015, 2016 University College London.
+
+This file is part of Nammu.
+
+Nammu is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Nammu is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Nammu.  If not, see <http://www.gnu.org/licenses/>.
+'''
+
+from ..view.EditSettingsView import EditSettingsView
+from ..utils import save_yaml_config
+
+
+class EditSettingsController:
+    def __init__(self, maincontroller):
+        self.controller = maincontroller
+        self.config = self.controller.config
+        self.load_config()
+        self.view = EditSettingsView(self, self.working_dir, self.servers,
+                                     self.keystrokes, self.languages,
+                                     self.projects)
+        self.view.display()
+
+    def load_config(self):
+        '''
+        The user's config file should containg all necessary information for
+        the settings editor.
```

Comment:
The configuration in the config file is for Nammu as a whole, but all the configuration bits needed by this specific class are in the config file. Does this make sense? I'll try to rephrase.